packages/manager/apps/support/index.js

```diff
@@ -0,0 +1,29 @@
+import support from '@ovh-ux/manager-support';
+import ngOvhOtrs from '@ovh-ux/ng-ovh-otrs';
+import uiRouter from '@uirouter/angularjs';
+import angular from 'angular';
+
+/* eslint-disable import/no-webpack-loader-syntax, import/extensions */
+import 'script-loader!jquery';
+import 'script-loader!lodash';
+/* eslint-enable import/no-webpack-loader-syntax, import/extensions */
+
+import {
+  APP_STATE,
+  APP_ACCOUNT_STATE,
+} from './index.routing';
+
+angular
+  .module('supportApp', [
+    ngOvhOtrs,
+    support,
+    uiRouter,
+  ])
+  .config(/* @ngInject */ ($stateProvider) => {
```

Comment:
```suggestion
  .config(routing);
```

and add at the top `import routing from './support.routing';`

Reply:
Why do we need to configure states to test the support module ?